### PR TITLE
Bugfix/kb 105

### PR DESCRIPTION
**Pull request by temaniarpit27**

### CODE REVIEW

File: crypto/keybase/keybase.go [hunk 1]
```diff
@@ -1,40 +1,29 @@
...
-	if !overwrite {
-		_, err := keybase.Get(keyName)
-		if err == nil {
-			return []byte{}, "", fmt.Errorf(
-				"key already exists, overwrite is disabled (clientRoot: %s)", clientRoot)
-		}
-	}
+	// TODO
+	// kn-sdk? appName
+	kr, err := keyring.New("kn-sdk", keyring.BackendPass, clientRoot, nil)
```
Comment (bldgk) [inline crypto/keybase/keybase.go]: kr may have nil if err is not nil

Comment (bldgk) [reply crypto/keybase/keybase.go]: There must be error checking code on the 15 line, before calling kr.NewMnemonic or kr.NewAccount functions

**Review by bldgk — CHANGES_REQUESTED** [review]

File: crypto/keybase/keybase.go (hunk 1 above)
Comment (bldgk) [inline crypto/keybase/keybase.go]: AppName - KonstellationApp

File: crypto/keybase/keybase.go [hunk 2]
```diff
@@ -1,40 +1,29 @@
...
 	if keyMnemonic == "" {
 		// generate a private key, with recovery phrase
-		info, keyMnemonic, err = keybase.CreateMnemonic(keyName, keys.English, keyPass, keys.Secp256k1)
+		info, keyMnemonic, err = kr.NewMnemonic(keyName, keyring.English, keyPass, hd.Secp256k1)
 	} else {
 		// account 0 "Account number for HD derivation"
 		// index 0 "Address index number for HD derivation"
-		info, err = keybase.CreateAccount(keyName, keyMnemonic, "", keyPass, 0, 0)
+		// TODO
+		// 118? coinType
```
Comment (bldgk) [inline crypto/keybase/keybase.go]: HDPath uses BIP44
coinType - 118, yes

**Review by bldgk — CHANGES_REQUESTED** [review]

**Comment by temaniarpit27** [comment]
@bldgk Please review and approve and merge

**Review by bldgk — APPROVED** [review]

**Pull request merged**